src/BinWrapper.jl

```diff
@@ -0,0 +1,161 @@
+kbmag_bin_dir   = joinpath(dirname(@__FILE__), "..", "deps", "usr", "bin", "")
+kbmag_data_dir  = joinpath(dirname(@__FILE__), "..", "deps", "src", "kbmag-1.5.8", "standalone", "kb_data")
```

Comment:
As indicated above, these directories are well-known to be there. Although I understand that putting any constants of this type in the heart of a package is not very... reasonable (?). What I suggest is to leave the `kbmag_bin_dir` as the constant (it will _not_ change) and assume that the execution dir is `.` by default. This way, the execution dir _should_ be declared as the parameter of all the functions as I did in `runtests.jl`.